diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -178,7 +178,9 @@ class BlockStructureBraces(private val configRules: List<RulesConfig>) : Rule("b
                 if (braceSpace == null || braceSpace.elementType != WHITE_SPACE) {
                     node.addChild(PsiWhiteSpaceImpl(" "), nodeBefore)
                 } else {
-                    (braceSpace as LeafPsiElement).replaceWithText(" ")
+                    val spaceParent = braceSpace.treeParent
+                    spaceParent.addChild(PsiWhiteSpaceImpl(" "), braceSpace)
+                    spaceParent.removeChild(braceSpace)
```

Comment:
Why does this fix an error? I think `replaceWithText` does more or less the same?